src/mscorlib/shared/System/Diagnostics/Tracing/EventProvider.cs

```diff
@@ -1166,6 +1167,7 @@ internal unsafe protected bool WriteEvent(ref EventDescriptor eventDescriptor, G
             status = m_eventProvider.EventWriteTransferWrapper(
                 m_regHandle,
                 ref eventDescriptor,
+                IntPtr.Zero,
```

Comment:
Because this `EventWriteTransferWrapper` is called by `WriteEventRaw` instead of `WriteEvent`. I don't think we care about `WriteEventRaw` at this point.